### DISCUSSION TITLE
Sources for menu options - Please share yours

### DISCUSSION THREAD

**Discussion by VA3HDL** [VA3HDL/hamdashboard#19] [General]
My personal menu options are:
```
"2196F3", "CAT", "../CAThtml/CAT_Frequencies.html", "1"

"2196F3", "HAMCLOCK", "http://192.168.0.111:8081/live.html", "1"

"2196F3", "PISTAR", "http://pi-star.local/", "1"

"2196F3", "ROTATOR", "../map/rotator.html", "1"

"FF3300", "APRS", "https://aprs.to/?center=43.9982,-79.4625&zoom=11", "1"

"FF3300", "BM", "https://brandmeister.network/", "1"

"FF3300", "CLUBLOG", "https://clublog.org/livestream/VA3HDL", "1.7"

"FF3300", "CONTEST", "https://www.contestcalendar.com/fivewkcal.html", "1"

"FF3300", "DX CLUSTER", "https://dxcluster.ha8tks.hu/map/", "1"

"FF3300", "HamDXmap", "https://dxmap.f5uii.net/", "1"

"FF3300", "PSKREPORTER", "https://pskreporter.info/pskmap.html?preset&callsign=VA3HDL&txrx=tx&timerange=86400&distunit=km&hidenight=1", "1"

"FF3300", "WINLINK", "https://cms.winlink.org:444/maps/propagation.aspx", "1"

"2196F3", "ADSB", "https://globe.adsbexchange.com/", "1", "R"

"2196F3", "LIGHTNING", "https://map.blitzortung.org/#3.87/36.5/-89.41", "1", "R"

"2196F3", "RADAR", "https://weather.gc.ca/?layers=alert,radar¢er=43.39961001,-78.53212031&zoom=6&alertTableFilterProv=ON", "1", "R"

"2196F3", "Stuff In Space", "https://stuffin.space/", "1", "R"

"2196F3", "TIME.IS", "https://time.is/", "1", "R"

"2196F3", "WEATHER", "https://openweathermap.org/weathermap?basemap=map&cities=true&layer=temperature&lat=44.0157&lon=-79.4591&zoom=5", "1", "R"

"2196F3", "WINDS", "https://earth.nullschool.net/#current/wind/surface/level/orthographic=-78.79,44.09,3000", "1", "R"
```

**Comment by WD4GAB** [comment]
["4660FF", "HamClock", "http://127.0.0.1:8081/live.html", "1"],

["4660FF", "Tides", "https://wave.marineweather.net/itide/tides/png/va_virginia_beach.png", "1",],

["2196F3", "DX CLUSTER", "https://dxcluster.ha8tks.hu/map/", "1"],

["2196F3", "LIGHTNING", "https://map.blitzortung.org/#3.87/36.5/-89.41", "1","R",],

["4660FF", "StarLink", "https://satellitemap.space/#", "1.2"],

["2196F3", "RADAR", "https://www.wunderground.com/wundermap?radar=1&wxstn=0&wxstnmode=tw", "1", "R"],

["2196F3", "TIME.IS", "https://time.is/", "1", "R"],

["4660FF", "Beacons", "https://www.reversebeacon.net/main.php?zoom=37.17,-11.73,2.05&rows=25&max_age=1,hours&bands=40,20,10,2&spot_types=1,2&hide=distance_km", "1", "L"],

["2196F3", "Weather", "https://openweathermap.org/weathermap?basemap=map&cities=true&layer=temperature&lat=44.0157&lon=-79.4591&zoom=5", "1", "R",],

["4660FF", "Wx Alerts", "https://alerts.weather.gov/search?area=VA", "1", "R",],

["2196F3", "Winds", "https://earth.nullschool.net/#current/wind/surface/level/orthographic=-78.79,44.09,3000", "1", "R",],

["4660FF", "Hurricanes", "https://www.nhc.noaa.gov/", "1", "R",],

["4460FF", "The Sky", "https://theskylive.com/planetarium?objects=sun-moon-mercury-venus-mars-jupiter-saturn-uranus-neptune-pluto&localdata=36.81904%7C-76.27494%7CChesapeake+VA+(US)%7CAmerica%2FNew_York%7C0&obj=sun&h=01&m=32&date=2024-07-21", "1", "R",],
["4460ff", "Propagation", "https://dxheat.com/dxc/", "1", "R"],
